### PR TITLE
fix(client): quick-fix for cert claiming

### PR DESCRIPTION
**Pull request by raisedadead**
This PR will

- reinstate the quick-fix for cert claiming Shaun made earlier (d80d093c8bd2ee9f24dace9e98c50eae9cdb7cfa)
- add progress to claim cert card
- link to the certificate's location in the settings
- should show the cards only to signed in and non-certified users.

image

This will be superseded by #41147

### CODE REVIEW

**Review by naomi-lgbt — DISMISSED** [review]
Looks good to me.

File: client/src/templates/Introduction/components/ClaimCertSteps.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  return createSelector(userSelector, user => ({
+    user
+  }))(state);
+};
+
+const ClaimCertSteps = ({ i18nCertText, superBlock, user }) => {
+  const { t } = useTranslation();
+
+  const settingsLink = '/settings#privacy-settings';
+  const certClaimLink = `/settings#settings-${superBlock}`;
```
Comment (ShaunSHamilton) [inline client/src/templates/Introduction/components/ClaimCertSteps.js:38]: The `certSlug` and the `superBlock` are not the same. So, this does not work for `-v7` certifications

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
Either, collect `superBlock` in `Certification.js` or collect `certSlug` in `ClaimCertSteps.js`. The latter is probably the easiest by passing down from `CertChallenge.js`.

**Review by ShaunSHamilton — APPROVED** [review]
LGTM 👍 These changes are quite nice.

**Review by naomi-lgbt — APPROVED** [review]
Looks good to me as well. A much nicer UX.

**Pull request merged**